can_tools/scrapers/official/ID/id_county.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import us
+
+from can_tools.scrapers.official.base import TableauDashboard
+from can_tools.scrapers import variables
+class IdahoCountyVaccine(TableauDashboard):
+    has_location = False
+    source = "https://coronavirus.idaho.gov/"
+    source_name = "Idaho Official Government Website"
+
+    baseurl = "https://public.tableau.com"
+    viewPath = "COVID-19VaccineDataDashboard/VaccineUptake"
+
+    state_fips = int(us.states.lookup("Idaho").fips)
+
+    variables = {
+        '1': variables.INITIATING_VACCINATIONS_ALL,
+        '2': variables.FULLY_VACCINATED_ALL
+    }
+
+    def fetch(self):
+
+        return self.get_tableau_view()
+    
+    def normalize(self, data):
+        df = data['Vax Rate / County Chart']
+        df.columns = [
```

Comment:
```suggestion
        data.columns = [
```
since the fetch method returns the specified table directly